apps/BraynsViewer/ui/Application.cpp

```diff
@@ -77,6 +79,8 @@ void glfwErrorCallback(int error, const char* description)
 }
 } // namespace
 
+using namespace brayns;
```

Comment:
Personal opinion, that might be problematic if new dependencies are added with similar names. It also make less clear from where certain types come from. But its not a big deal, so no need to change it.

Reply:
Ah yeah actually I added that temporary but I forgot to remove it, thanks.